src/components/home/HomePage.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div
```

Comment:
While we won't be using it elsewhere, I think this could already be it's own component.
Not for re-usability, but to keep this homepage template from getting too big down the line.